umh.docs.umh.app/content/en/docs/features/data-connectivity-node-red/index.md

```diff
@@ -3,31 +3,78 @@ title = "Data connectivity with Node-RED"
+- connect to serial (node-red-node-serialport)
+- connect to modbus (node-red-contrib-modbus had)
+- connect to MC-protocol (node-red-contrib-mcprotocol)
+- connect to OMRON FINS Ethernet protocol (node-red-contrib-omron-fins)
+- connect to EtherNet/IP Protocol (node-red-contrib-cip-ethernet-ip)
+- connect to PostgreSQL (node-red-contrib-postgresql)
+- connect to SAP SQL Anywhere
+
+You can additionally contextualize the data, using function or other different nodes do manipulate the 
+received data.
 
 ## How can I use it?
 
-See also tutorial on accessing Node-RED
+Node-RED comes preinstalled as a microservice with the United Manufacturing Hub.
+
+1. To access Node-RED navigate to **network**->**services** on the left-hand side in UMHLens. You can download UMHLens [here](https://github.com/united-manufacturing-hub/UMHLens/releases).
+2. Change the namespace to **Namespace: united-manufacturing-hub** at the top right.
+
+   ![Untitled](/images/getstarted/managingTheSystem/getStartedManagingServices.png?width=75%)
+
+3. Click on **united-manufacturing-hub-nodered-service** and scroll down to **Connection** and forward the port.
+
+   ![Untitled](/images/getstarted/managingTheSystem/getStartedManagingForwarding.png?width=75%)
+4. Once Node-RED opens in the browser, add `nodered` to the URL to avoid the [cannot get error](https://learn.umh.app/course/how-to-fix-cannot-get-error-in-node-red/).
+5. Begin exploring right away! If you require inspiration on where to start, we provide a variety of guides to help you
+      become familiar with various node-red workflows, including how to process data and align it with the UMH datamodel:
+
+    - [Create a Node-RED Flow with Simulated OPC-UA Data](https://learn.umh.app/course/creating-a-node-red-flow-with-simulated-opc-ua-data/)
+    - [Create a Node-RED flow with simulated PackML data](https://learn.umh.app/course/creating-a-node-red-flow-with-packml/)
+    - Alternatively, visit [learning page](https://learn.umh.app/topic/node-red/) where you can find multiple best practices for using Node-RED
+
+
 
 ## What are the limitations?
 
-- most packages have no enterprise support. If you encounter any errors, you need to ask the community. However, we found that these packages are often more stable than the commercial ones out there, as they have been battle tested by way more users than commercial software
-- 
+- most packages have no enterprise support. If you encounter any errors, you need to ask the community. 
+  However, we found that these packages are often more stable than the commercial ones out there, 
+  as they have been battle tested by way more users than commercial software.
+- having many flows without following a strict structure, leads in general to confusion.
+- one additional limitation is "the speed of development of Node-RED". After a big Node-RED and JavaScript update
+  dependencies most likely break, and those single community maintained nodes need to be updated.
```

Comment:
Capitalize first letter.
Last point is a bit confusing, what do you want to say there?